src/components/NavBar/AccountMenu.tsx

```diff
@@ -1,18 +1,35 @@
 import { useAuth0 } from "@auth0/auth0-react";
-import { Avatar, Link } from "@mui/material";
+import { Avatar, Box, Link } from "@mui/material";
 import { Typography } from "@mui/material";
 import { Button } from "@mui/material";
 import Popover from "@mui/material/Popover";
 import Stack from "@mui/material/Stack/Stack";
 import React from "react";
 import { useAppSelector } from "../../state/hooks";
+import LogoutIcon from "@mui/icons-material/Logout";
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
 
 const styles = {
   button: {
     marginTop: "auto",
     marginBottom: 2,
   },
   nameContainer: { paddingLeft: 2, textAlign: "left" },
+  popover: {
+    paddingTop: "10px",
+    paddingBottom: "10px",
+    borderRadius: "10px",
+  },
+  popoverButton: {
+    paddingTop: "5px",
+    paddingBottom: "5px",
+    paddingRight: "20px",
+    paddingLeft: "20px",
```

Comment:
Same here, please use numbers. If we also shift some of the padding to the pop over component we can reduce the padding on the buttons. Also a cool thing with MUI is that they give us shortcuts for things like adding padding only to the horizonal axis; we can use `paddingX` and `paddingY` to add padding to only the X axis and Y axis respectively (the same applies to margin). If we follow the padding suggestion for the popover I mentioned earlier I think we can do something like:
```suggestion
    paddingX: 2
```

Reply:
Thanks for this, I see a bunch more suggestions and shortcuts in the MUI spacing documentation. Regarding the suggestions, I have updated it.